Makefile

```diff
@@ -314,7 +314,7 @@ endif
 
 test-javascript:
 	$(call announce-begin,"Running JavaScript tests")
-	npm install; \
+	npm install
```

Comment:
With the `\` I guess make treats `npm install; npm test` as one command. Without it, `npm install` fails properly

Reply:
@silverjam should I bump serialport now that we know it fails properly?

Reply:
@notoriaga please, we should refresh the javascript stuff to whatever makes sense for modern JS development